### PR TITLE
 add std format 

### PR DESCRIPTION
**Pull request by damody**
add std format! marco.

### CODE REVIEW

File: src/format_bench.rs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            table_str.push_str(format!("<td>{col}</td>",col = r2).as_str());
+        }
+        table_str.push_str("</tr>\n");
+    }
+    let output = format!(
+"<table>
+{table_str}
+</table>", table_str = table_str);
+
+    b.iter(|| output.to_owned());
```
Comment (djc) [inline src/format_bench.rs]: This does not actually benchmark the `format!()` macro. For that to happen, I think you have to make the construction of `table_str` (so line 13-24) part of the closure passed to `b.iter()`.

File: src/format_bench.rs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <title>{year}</title>
+  </head>
+  <body>
+    <h1>CSL {year}</h1>
+    <ul>
+    {team_str}
+    </ul>
+  </body>
+</html>", year = teams.year, team_str = team_str);
+    b.iter(|| output.to_owned());
```
Comment (djc) [inline src/format_bench.rs]: Same problem here.

**Comment by djc** [comment]
@damody hey, so what's the plan? Will you submit a new one?

**Comment by damody** [comment]
- I write write! and fomat! you can see my fork.
- https://github.com/damody/template-benchmarks-rs
- Because I have no idea about avoid tmp variable.

**Pull request closed without merging**